build/gulpfile.editor.js

```diff
@@ -235,3 +235,60 @@ function filterStream(testFunc) {
 		this.emit('data', data);
 	});
 }
+
+
+//#region monaco type checking
+
+function createTscCompileTask(watch) {
+	return () => {
+		const createReporter = require('./lib/reporter').createReporter;
+
+		return new Promise((resolve, reject) => {
+			const args = ['./node_modules/.bin/tsc', '-p', './src/tsconfig.monaco.json', '--noEmit'];
+			if (watch) {
+				args.push('-w');
+			}
+			const child = cp.spawn(`node`, args, {
```

Comment:
Maybe use `cp.fork`, in order to get the same node instance?

Reply:
Yeah, can give it a try